pkg/vpcmodel/abstractVPC.go

```diff
@@ -209,6 +209,8 @@ type VPC interface {
 type Subnet interface {
 	NodeSet
 	CIDR() string
+	IsPublic() bool
+	SetIsPublic(bool)
```

Comment:
is this relevant as part of the abstract interface?

Reply:
It is an aws concept, However, I thought that it will be needed at explainability (and drawio).
Now, I am not sure that these functions are the right interface for explainability.
I added support for drawio, and see and eventually they are not needed for drawio.
so I deleted the functions from the interface, for now.
see my fix